crates/pallet-humanode-offences/src/mock.rs

```diff
@@ -35,6 +35,7 @@ frame_support::construct_runtime!(
         System: frame_system,
         Bootnodes: pallet_bootnodes,
         Bioauth: pallet_bioauth,
+        FixedValidatorsSet: pallet_fixed_validators_set,
```

Comment:
What about adding tests to verify slashing logic for identification related to `FixedValidatorSet`?

Reply:
Right, there should be no slashing; there aren't any tests that I could modify/extend to add this, are there?